src/main/java/ladder/domain/result/Result.java

```diff
@@ -0,0 +1,77 @@
+package ladder.domain.result;
+
+import ladder.domain.ladder.Block;
+import ladder.domain.ladder.Ladder;
+import ladder.domain.ladder.Line;
+import ladder.domain.player.PlayerName;
+import ladder.domain.player.Players;
+import ladder.domain.prize.Prize;
+import ladder.domain.prize.Prizes;
+import ladder.domain.player.exception.NoSuchPlayerException;
+
+import java.util.*;
+
+public class Result {
+
+    private final Map<PlayerName, Prize> results;
+
+    public Result(final Players players, final Prizes prizes, final Ladder ladder) {
+        this.results = makeResult(players, prizes, ladder);
+    }
+
+    private Map<PlayerName, Prize> makeResult(final Players players, final Prizes prizes, final Ladder ladder) {
+        List<PlayerName> playerNames = new ArrayList<>(players.getPlayers());
+        ladder.getLines().forEach(line -> moveOneLine(playerNames, line));
```

Comment:
`.` 단위로 줄바꿈을 하면 읽기도 편하고 나중에 디버깅할 때도 더 좋답니다~

```suggestion
ladder.getLines()
      .forEach(line -> moveOneLine(playerNames, line));
```